examples/tests/ieee802154/radio_tx_direct/main.c

```diff
@@ -0,0 +1,40 @@
+#include <stdbool.h>
+#include <stdio.h>
+
+#include "gpio.h"
+#include "ieee802154.h"
+#include "led.h"
+#include "timer.h"
+#include "tock.h"
+
+// IEEE 802.15.4 sample packet transmission app.
+// Continually transmits frames at the specified short address to the specified
+// destination address.
+
+#define BUF_SIZE 3
+char packet[BUF_SIZE] = {2, 0, 1};
```

Comment:
Is this a valid and meaningful 15.4 packet? It would be nice for it to be.

Reply:
Yes, this is a simple ACK packet. I will update this to have a comment mentioning this.